eva_cttv_pipeline/trait_mapping/utils.py

```diff
@@ -0,0 +1,18 @@
+def request_retry_helper(function, retry_count: int, url: str):
```

Comment:
The OxO function is a bit more complex because it takes a payload because the function it calls uses a post request, rather than a get request like in the case of this function.